app/models/groupe_instructeur.rb

```diff
@@ -59,5 +59,15 @@ def can_delete?
 
   def toggle_routing
     procedure.update!(routing_enabled: procedure.groupe_instructeurs.active.many?)
+
+    if procedure.groupe_instructeurs.active.size == 2 && !procedure.has_a_tdc_routage?
+      procedure.draft_revision.add_type_de_champ(
+        type_champ: 'routage',
+        libelle: procedure.routing_criteria_name || procedure.routing_libelle,
```

Comment:
s'il n'a pas de tdc_routage, il n'a pas de routing_libelle,
```suggestion
        libelle: procedure.routing_criteria_name
```